tag: 'iris'

anchors: [
  'anchor',
  'anchor_0'
]

topic_prefix: '/uwb'
update_rate: 15